### ISSUE TITLE
delete fails on a MediaInstance but works on MediaContext

### ISSUE THREAD

**Issue by markquezada** [twilio/twilio-ruby#197]
Using 5.0.0.rc4:

Not sure if I'm doing something wrong or if this is expected:

**MediaContext**

```ruby
media_context = $twilio.account.messages(@message_sid).media(@medium_sid)
media_context.delete
=> true
```

**MediaInstance**

```ruby
media_instance = $twilio.account.messages(@message_sid).media(@medium_sid).fetch
media_instance.delete
=> NoMethodError: undefined method `delete' for nil:NilClass
```

**Comment by carlosdp** [comment]
Fixed in 7fe6b82

Released as 5.0.0.rc10

**Comment by markquezada** [comment]
Just tested and it works. Thanks for fixing this!
